bittide-instances/src/Bittide/Instances/Tests/FullMeshHwCc.hs

```diff
@@ -163,8 +190,8 @@ goFullMeshHwCcTest refClk sysClk rst rxns rxps miso =
   fincFdecIla :: Signal Basic125 ()
   fincFdecIla = ila
     (ilaConfig $
-         "trigger"
-      :> "capture"
+         "trigger_0"
+      :> "capture_0"
```

Comment:
How come? Does the TCL accept this?

Reply:
I suppose you mean the suffix of the probe names, in which case yes. The TCL accepts any probe with the name `*/trigger*`, as long as there is only one. The same goes for `/capture*`.

Reply:
We need this because of the other ILA. Vivado doesn't like if they share names.